### DISCUSSION TITLE
Single dockerfile for production

### DISCUSSION THREAD

**Discussion by Romain-Dfs** [dunglas/symfony-docker#265] [Q&A]
Hi !

I'm trying to create a single Dockerfile to upload to Clever Cloud
However, I can't manage to do it and I wanted to know if it's possible at first, and if so, if it's a good practice

Wishing you an excellent day,
Sincerely

**Comment by maxhelias** [comment]
Hi! 

I hope this will help you : https://medium.com/@stoyanov.veseline/pushing-docker-images-to-a-private-registry-with-docker-compose-d2797097751
